### ISSUE TITLE
v3: bucket list of items

### ISSUE THREAD

**Issue by xenowits** [xenowits/moneyprinter.info#12]
## Problem to be solved

We need to plan items to be included as part of v3.

## Proposed solution

* Build a multi-page application instead.
* We can't introduce a lot of data in the current single page application.
* Add data for controlling percentages as `51%` and `66%` in addition to the `33.33%` we currently have.
* Add a new page to list cosmos app-chains.
* Add another metric to track operators controlling large stakes in multiple different networks.
* In other words, operators which are a part of nakamoto coefficient for different networks simultaneously.
* Changes in data layout
* More data representations, preferably showing a history of nakamoto coefficient changes
